### PR TITLE
MNIST Digit Classification using Neural Networks

### PR DESCRIPTION
**Pull request by Cgarg9**
# Pull Request for DL-Simplified 💡

## Issue Title : MNIST Digit Classification using Neural Networks

- **Info about the related issue (Aim of the project)** : To classify digits in the MNIST dataset using neural networks.
- **Name:**:  Chirag Garg
- **GitHub ID:** : https://github.com/Cgarg9
- **Email ID:** : chiraggarg.red@gmail.com
- **Idenitfy yourself: Codepeak23

Closes: #issue number that will be closed through this PR
315 https://github.com/abhisheks008/DL-Simplified/issues/315
### Describe the add-ons or changes you've made 📃
I have added two models to classify the digits. Model 1) Artificial Neural Network, Model 2) Convolutional Neural Network
Give a clear description of what have you added or modifications made
I have added two models to classify the digits. Model 1) Artificial Neural Network, Model 2) Convolutional Neural Network. These networks were trained for 10 epochs and both managed to achieve an accuracy of about 99%.
## Type of change ☑️

What sort of change have you made:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ⚙️
Describe how it has been tested
A testing dataset is provided by MNIST and the code was tested against it. 
Describe how have you verified the changes made
All errors faced have been resolved using the guidance from various online resources and official documentations of the libraries used.
## Checklist: ☑️
- [ x] My code follows the guidelines of this project.
- [ x] I have performed a self-review of my own code.
- [ x] I have commented my code, particularly wherever it was hard to understand.
- [ x] I have made corresponding changes to the documentation.
- [ x] My changes generate no new warnings.
- [ x] I have added things that prove my fix is effective or that my feature works.
- [ x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
1. Please add EDA in your project to find out various visualizations from the dataset.
2. Follow the `README.md` template and update accordingly.
3. Your Jupyter notebook's cells are not showing the output properly, please take a look at those.

@Cgarg9

**Comment by abhisheks008** [comment]
Hi @Cgarg9 any update on the requested changes?

**Comment by Cgarg9** [comment]
Almost done. Will make the pr by tomorrow
